index.html

```diff
@@ -54,24 +54,58 @@
     </script>
   </head>
 
-  <body>
+  <body class="main-body">
     <!--header-->
     <header class="toolbar d-flex align-center">
       <a href="#" class="logo">Elikplim</a>
-      <nav class="navbar">
-        <i class="fa fa-bars bar-icon"></i>
+      <nav class="navbar-desktop">
         <ul>
           <li>Portfolio</li>
           <li>About</li>
           <li>Contact</li>
           <li><i class="fa fa-envelope"></i></li>
         </ul>
       </nav>
+
+      <div class="bar-icon">
+        <i class="fa fa-bars"></i>
+      </div>
+
+      <!--mobile menu -->
```

Comment:
- [ ] Let's bring the margins/padding closer to the Figma template:

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/57421823/152020128-9e300f7f-5540-4dd2-b981-d442ba7fbddd.png)| ![image](https://user-images.githubusercontent.com/57421823/152020226-a53e4c10-2b3c-4a03-858e-c42f04790c13.png)

As you can see, we need to add some margins/padding to bring the menu items and the close button a little lower, so the design ends up matching the Figma.

Also here:

- [ ] The arrows at the end of each menu item please kindly make them semi-transparent, like in the Figma.
- [ ] Make the close button a little bigger, so it matches the Figma template.

Look:

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/57421823/152021209-e07bd12a-b29f-442b-9049-799e639380e9.png) | ![image](https://user-images.githubusercontent.com/57421823/152021431-9c5d5c8c-246e-43f7-817b-0ff38511ff5e.png)


That way, your mobile menu will look striking!! 💪 💪

Reply:
Addressed ✔